### ISSUE TITLE
JSON flag gives incorrect line/column numbers

### ISSUE THREAD

**Issue by binarycleric** [amperser/proselint#418]
When I run `proselint` with only a filepath, any errors I am shown display the correct line and column.

```bash
proselint ✔ $ proselint app/scholarship/scholarship.md
app/scholarship/scholarship.md:7:56: butterick.symbols.multiplication_symbol Use the multiplication symbol ×, not the letter x.
````

However, setting the `--json` gives me results that are 1 line/column off.

```
proselint ✔ $ proselint --json app/scholarship/scholarship.md | jq
{
  "status": "success",
  "data": {
    "errors": [
      {
        "start": 583,
        "replacements": null,
        "end": 586,
        "severity": "warning",
        "extent": 3,
        "column": 55,
        "message": "Use the multiplication symbol ×, not the letter x.",
        "line": 6,
        "check": "butterick.symbols.multiplication_symbol"
      }
    ]
  }
}
```

I've tried numerous markdown files and am seeing this across the board. Let me know if I can do anything else to help.

**Comment by suchow** [comment]
I think the question we need to ask is whether it's conventional to use zero-based numbering or start at one. Let's see what `pep8` does:

```python
x=1
```

```shell
❯ pep8 test.py
test.py:1:2: E225 missing whitespace around operator
```

Okay, so it is indexed starting at one.

Good catch, thanks.

**Comment by suchow** [comment]
Closed by #429.
